docs/learning-powershell/README.md

```diff
@@ -85,7 +85,7 @@ Note that all bash commands should continue working on PowerShell session.
 | Bash                | PowerShell                  | Description
 |:--------------------|:----------------------------|:---------------------
 | ls                  |dir, Get-ChildItem           |List files and folders
-| tree                |dir -Recurse                 |List all files and folders
+| tree                |dir/Get-ChildItem -Recurse   |List all files and folders
```

Comment:
Maybe:
```suggestion
| tree                |dir -Recurse, Get-ChildItem -Recurse   |List all files and folders
```

Reply:
I was my original idea (represent both commands), but I recognize that this example is best.